clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/DataRecoveryClient.java

```diff
@@ -45,8 +52,35 @@ public DataRecoveryMonitor getMonitor() {
     return monitor;
   }
 
+  public void filterStoresWithOngoingRepush(Set<String> storesList, StoreRepushCommand.Params params) {
+    String url = params.getUrl();
+    ControllerClient cli = params.getPCtrlCliWithoutCluster();
+    LocalDateTime timestamp = params.getTimestamp();
+    String sourceFabric = params.getSourceFabric();
+    for (String s: storesList) {
+      String clusterName = cli.discoverCluster(s).getCluster();
+      try (ControllerClient parentCtrlCli = new ControllerClient(clusterName, url, Optional.empty())) {
+        StoreHealthAuditResponse storeHealthInfo = parentCtrlCli.listStorePushInfo(s, false);
+        RegionPushDetails regionPushDetails = storeHealthInfo.getRegionPushDetails().get(sourceFabric);
```

Comment:
I don't think we care about source fabric. Instead, dest fabric is what we should check for.